src/client/sky.cpp

```diff
@@ -161,7 +160,8 @@ void Sky::render()
 		float wn = nightlength / 2;
 		float wicked_time_of_day = 0;
 		if (m_time_of_day > wn && m_time_of_day < 1.0 - wn)
-			wicked_time_of_day = (m_time_of_day - wn) / (1.0 - wn * 2) * 0.5 + 0.25;
+			wicked_time_of_day = (m_time_of_day - wn) / (1.0 - wn * 2) * 0.5 +
+					     0.25;
```

Comment:
Lines only need splitting if over 90 columns (with tab size 4).
If <=90 columns the previous format was better.